comfy_extras/nodes_video_model.py

```diff
@@ -33,22 +33,30 @@ def INPUT_TYPES(s):
                               "video_frames": ("INT", {"default": 14, "min": 1, "max": 4096}),
                               "motion_bucket_id": ("INT", {"default": 127, "min": 1, "max": 1023}),
                               "fps": ("INT", {"default": 6, "min": 1, "max": 1024}),
-                              "augmentation_level": ("FLOAT", {"default": 0.0, "min": 0.0, "max": 10.0, "step": 0.01})
-                             }}
+                              "augmentation_level": ("FLOAT", {"default": 0.0, "min": 0.0, "max": 10.0, "step": 0.01}),
+                    },
+                "optional": { "noise_seed": ("INT", {"default": 0, "min": 0, "max": 2**32-1})
```

Comment:
the seed max value as seen in eg the KSampler node should be set as `0xffffffffffffffff`